packages/suite/src/views/wallet/tokens/common/TokensTable/TokenRow.tsx

```diff
@@ -346,6 +421,33 @@ export const TokenRow = ({
                             ].filter(category => category) as GroupedMenuItems[]
                         }
                     />
+                    {!isMobileLayout && (
+                        <IconButton
+                            label={
+                                canSwapToken ? (
+                                    <Translation id="TR_COINMARKET_SWAP_UNAVAILABLE" />
+                                ) : (
+                                    <Translation id="TR_COINMARKET_SWAP" />
+                                )
+                            }
+                            isDisabled={canSwapToken}
```

Comment:
```suggestion
                                !canSwapToken ? (
                                    <Translation id="TR_COINMARKET_SWAP_UNAVAILABLE" />
                                ) : (
                                    <Translation id="TR_COINMARKET_SWAP" />
                                )
                            }
                            isDisabled={!canSwapToken}
```
it is exactly the other way around. I will fix that 😀

Reply:
🫣gracia!